default/inputs.conf

```diff
@@ -1,2 +1,4 @@
 [jwtwebhook]
 port=4443
+cert_file = etc/auth/splunkweb/cert.pem
```

Comment:
i don't think we should use splunkweb's certificate as default. @acharlieh thoughts?

Reply:
It doesn’t need to be default, but have to allow the package to be useable from the moment a customer decides to use it. Otherwise, we should provide instructions on the available options the customer can pick:
1. Manual Self signed
2. Use OOTB default Splunk self-signed
3. Signed cert with

Reply:
Given that this is going to run on an HF, opening a listening HTTPS port by default... we should pick either the default splunkweb cert&key or the SplunkAPI cert&key as default, and an admin can choose to replace if need be. (and the nice thing of the default splunkweb key is there is no keyfile password by default out of the box.)

This way those who don't have the security concerns and want to get up and going quickly (e.g. demo systems) can do so without much forced configuration. I would say 

1. We should note this default in the README.md and recommend that secure systems change this cert
1. Should the values be starting `$SPLUNK_HOME/` ? or does a relative path default to being relative to Splunk Home?

Reply:
Confirmed and validated with @acharlieh. These default will work and the TA will automatically update the inputs.conf either from the UI or backend. We will adopt a better approach in the next "release".